### PR TITLE
prepared chk1 and hsp90_kung in waterset

### PR DESCRIPTION
**Pull request by vgapsys**
## Input submission checklist

* [ x] created a directory for each system based on the [template directory][templates]
* [ x] named and placed the directory with the following pattern: `input_structures/prepared_structures/<set_name>/<system_name>`

For each submitted directory:

* [x] filled in system preparation details in the `PREPARATION_DETAILS.md` file
* [x] added a PDB file with the protein named `protein.pdb`
* [x] removed any cofactors from the PDB file and placed them in `cofactors.sdf`
* [x] did not re-protonate the residues, but left protonation state of the protein unchanged
* [x] kept cystallographic waters and metals in the PDB file
* [x] tested the PDB and, if available, cofactors.sdf file using the [input validation script][input_validation]
* [x] copied over the ligands SDF file to a file named `ligands.sdf` and removed any duplicate binding modes or protonation states, keeping the more favorable state

## Licensing agreement
* [x] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

[templates]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks
[input_validation]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/input_validation.py

### CODE REVIEW

File: industry_benchmarks/input_structures/prepared_structures/water_set/hsp90_kung/PREPARATION_DEATILS.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+protein:
+ - used maestro protein preparation to cap N-term and C-term (in the original structure termini are neutral and uncapped)
```
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/water_set/hsp90_kung/PREPARATION_DEATILS.md:3]: Could you add the maestro version used here?

**Review by hannahbaumann — CHANGES_REQUESTED** [review]
Thanks @vgapsys , just the small comment about the maestro version, otherwise looks good to me!

File: industry_benchmarks/input_structures/prepared_structures/water_set/hsp90_kung/PREPARATION_DEATILS.md (hunk 1 above)
Comment (vgapsys) [reply industry_benchmarks/input_structures/prepared_structures/water_set/hsp90_kung/PREPARATION_DEATILS.md]: done

**Review by IAlibay — APPROVED** [review]
Thanks @vgapsys :)

**Review by hannahbaumann — APPROVED** [review]

**Pull request merged**